Source/CesiumRuntime/Private/CesiumDmsEditor.cpp

```diff
@@ -0,0 +1,290 @@
+  // or "North" or "South", respectively.
+  if (IsLongitude) {
+    PositiveIndicator = MakeShareable(new FString(TEXT("E")));
+    NegativeIndicator = MakeShareable(new FString(TEXT("W")));
+  } else {
+    PositiveIndicator = MakeShareable(new FString(TEXT("N")));
+    NegativeIndicator = MakeShareable(new FString(TEXT("S")));
+  }
+  SignComboBoxItems.Add(NegativeIndicator);
+  SignComboBoxItems.Emplace(PositiveIndicator);
+  SignComboBox = SNew(STextComboBox)
+                     .OptionsSource(&SignComboBoxItems)
+                     .OnSelectionChanged(this, &CesiumDmsEditor::SignChanged);
+  SignComboBox->SetSelectedItem(
+      GetDecimalDegreesFromProperty() < 0 ? NegativeIndicator
+                                          : PositiveIndicator);
+
+  const float hPad = 3.0;
+  const float vPad = 2.0;
+  // clang-format off
+	Row.CustomWidget().NameContent()
+		[
+			DecimalDegreesHandle->CreatePropertyNameWidget()
+		]
+		.ValueContent().HAlign(EHorizontalAlignment::HAlign_Fill)
+		[
+        SNew( SVerticalBox )
+        + SVerticalBox::Slot().Padding(0.0f, vPad)
+        [
+              DecimalDegreesSpinBox.ToSharedRef()
+        ]
+        + SVerticalBox::Slot().Padding(0.0f, vPad)
+        [
+            SNew( SHorizontalBox )
+            + SHorizontalBox::Slot().FillWidth(1.0)
+            [
+                DegreesSpinBox.ToSharedRef()
+            ]
+            + SHorizontalBox::Slot().AutoWidth().Padding(hPad, 0.0f)
+            [
+                // The 'degrees' symbol
+                SNew(STextBlock).Text(FText::FromString(TEXT("\u00B0")))
+            ]
+            + SHorizontalBox::Slot().FillWidth(1.0)
+            [
+                MinutesSpinBox.ToSharedRef()
+            ]
+            + SHorizontalBox::Slot().AutoWidth().Padding(hPad, 0.0f)
+            [
+                // The 'minutes' symbol
+                SNew(STextBlock).Text(FText::FromString(TEXT("\u2032")))
+            ]
+            + SHorizontalBox::Slot().FillWidth(1.0)
+            [
+                SecondsSpinBox.ToSharedRef()
+            ]
+            + SHorizontalBox::Slot().AutoWidth().Padding(hPad, 0.0f)
+            [
+                // The 'seconds' symbol
+                SNew(STextBlock).Text(FText::FromString(TEXT("\u2033")))
+            ]
+            + SHorizontalBox::Slot().AutoWidth()
+            [
+                SignComboBox.ToSharedRef()
+            ]
+        ]
+		];
+  // clang-format on
+}
+
+double CesiumDmsEditor::GetDecimalDegreesFromProperty() const {
+  double decimalDegrees;
+  FPropertyAccess::Result AccessResult =
+      DecimalDegreesHandle->GetValue(decimalDegrees);
+  if (AccessResult == FPropertyAccess::Success) {
+    return decimalDegrees;
+  }
+  // This should never be the case, if the actual prperty
+  // was a double property, so this warning indicates a
+  // developer error:
+  UE_LOG(LogCesium, Warning, TEXT("GetDecimalDegreesFromProperty FAILED"));
```

Comment:
Good use for an assertion, then. Especially if we're not going to do anything different when we detect this case, but just return decimalDegrees as we normally would.